src/main/java/nl/tudelft/contextproject/model/entities/EntityType.java

```diff
@@ -21,12 +21,13 @@
 	LANDMINE("LandMine", 			7,  LandMine::loadEntity),
 	
 	CARROT("Carrot", 				8,  Carrot::loadEntity),
-	KILLER_BUNNY("KillerBunny", 	9,  KillerBunny::loadEntity),
+	KILLER_BUNNY("KillerBunny", 	9,  null),
```

Comment:
Loading a bunny requires you to have a player in the game, but this player is created after the level is loaded. Therefore i decided to remove the possibility to load the bunny. @Taeir, however, told me a way to fix this, so I will try to fix it.